media/js/firefox/accounts-features.js

```diff
@@ -0,0 +1,10 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function(Mozilla) {
+    'use strict';
+
+    Mozilla.FxaState.getStateClassAndDo(Mozilla.FxaState.applyStateToBody);
+
+})(window.Mozilla);
```

Comment:
Yes absolutely! I wiffed on this because I couldn't figure out what to call it and I was sick of looking at the PR. Thank you for naming the thing for me. 😆